credits/zh-cn.json

```diff
@@ -0,0 +1,16 @@
+{
+    "developer": "开发",
```

Comment:
I cannot confirm from translations, is this the right term for software developer ?

Reply:
"Software Developer" is "软件开发" in Chinese. For gaming, we usually use "程序开发/Programmers" or the single word "开发/Developers" for the programming part.

I think it can reveal enough meaning for this part.

Reply:
Okay, thanks for the explanation, we can keep it like that then.